src/cmr/concepts/variable.js

```diff
@@ -27,6 +30,49 @@ export default class Variable extends Concept {
     return items
   }
 
+  /**
+   * Parses the response from the json endpoint
+   * @param {Object} jsonResponse HTTP response from the CMR endpoint
+   * @param {Array} jsonKeys Array of the keys requested in the query
+   */
+  async parseJson(jsonResponse, jsonKeys) {
+    const { headers } = jsonResponse
+    const {
+      'cmr-hits': cmrHits,
+      'cmr-search-after': jsonSearchAfterIdentifier
+    } = downcaseKeys(headers)
+
+    this.setJsonItemCount(cmrHits)
+
+    this.setJsonSearchAfter(jsonSearchAfterIdentifier)
+
+    const items = this.parseJsonBody(jsonResponse)
+
+    items.forEach((item) => {
+      const normalizedItem = this.normalizeJsonItem(item)
+
+      const { concept_id: conceptId } = normalizedItem
+      this.setEssentialJsonValues(conceptId, normalizedItem)
+
+      jsonKeys.forEach((jsonKey) => {
+        // Snake case the key requested and any children of that key
+        const cmrKey = snakeCase(jsonKey)
+
+        const { [cmrKey]: keyValue } = normalizedItem
+
+        let formattedKeyValue = keyValue
+
+        // Once accessed, ensure that the child key/value pairs are consistently returning in `camelCase`
+        // Note applying camelcase logic to all keys in `concepts` will not work because of `tags` business logic
+        if (jsonKey === 'scienceKeywords' || jsonKey === 'instanceInformation') {
```

Comment:
If the only field we know of that we don't want camelCased is 'tags', then use `!== 'tags'`, that will ensure future fields that need to follow the naming scheme will match without needing more conditionals.

That logic also seems like it should be uniform across all concepts. Is this line the only thing different between concept.js and variable.js? If both those things are true this logic should be in concept.js

Reply:
Yea I did it that way initially. It is also going to be fields within the `data` inside of the association-details which work the same way. Yes that is the only difference it is simple to add this to concepts instead